src/main/java/org/elasticsearch/zeromq/ZMQRestImpl.java

```diff
@@ -81,7 +80,7 @@ private ZMQRestResponse convert(RestResponse response) throws IOException {
 			zmqResponse.setContentType(response.contentType());
 		}
         if (response.content().length() > 0) {
-            if (response.contentThreadSafe()) {
+            if (zmqResponse.contentThreadSafe()) {
```

Comment:
response.contentThreadSafe() function does not exist. So I had to use zmqRespose and that breaks the unit tests.